src/utils/generateId.js

```diff
@@ -0,0 +1,9 @@
+const generateId = function (array) {
+  if (array.length > 0) {
```

Comment:
Can also be done like this:
```
const generateId = function (array) {
    return Number((array.at(-1) ?? { id: -1 }).id) + 1;
}
```
This tries to get the last element from the array `array.charAt(-1)`. If the array is empty, this method returns `undefined`.
When `undefined`, the fallback returns an object with the id "-1": `?? { id: -1 }`.
Then, it takes the id from that result (either the id from the last element, either "-1".
After that, it converts this id to a number `Number(...)`.
And finally, adds 1 to it `+ 1`.

If the array is empty, an object with id = -1 will be returned, -1 with + 1 returns 0 for the initial id of the database, and adds as should for all others